src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -301,12 +300,13 @@ def _flexible_server_params(command_group):
             c.argument('tags', tags_type)
             if command_group == 'mysql':
                 c.argument('tier', options_list=['--tier'],
-                           help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized ')
+                           help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized')
                 c.argument('sku_name', options_list=['--sku-name'],
-                           help='The name of the compute SKU. Follows the convention Standard_{VM name}. Examples: Standard_B1ms, Standard_D4s_v3 ')
+                           help='The name of the compute SKU. Follows the convention Standard_{VM name}. Examples: Standard_B1ms, Standard_E16ds_v4 ')
                 c.argument('storage_mb', options_list=['--storage-size'], type=int,
-                           validator=mysql_storage_validator,
                            help='The storage capacity of the server. Minimum is 5 GiB and increases in 1 GiB increments. Max is 16 TiB.')
+                c.argument('backup_retention', type=int, options_list=['--backup-retention'],
```

Comment:
Here is to define the retention days but how could I enable/disable delete retention?

Reply:
It cannot be disabled. By default it is 7 days. But user can pass a value and override the default value.